com.sap.cloud.lm.sl.cf.api/src/main/java/com/sap/cloud/lm/sl/cf/web/api/model/Message.java

```diff
@@ -19,7 +19,7 @@ public Message id(Long id) {
         return this;
     }
 
-    @ApiModelProperty(value = "")
```

Comment:
This is generated code. Avoid touching it.

Reply:
api model property has a default value for "value" which is this empty string.
That's why I removed it